shell/platform/darwin/ios/framework/Source/FlutterViewControllerTest.mm

```diff
@@ -67,68 +68,62 @@ - (void)dispatchPresses:(NSSet<UIPress*>*)presses;
 
 @implementation FlutterViewControllerTest
 
+- (void)setUp {
+  self.mockEngine = OCMClassMock([FlutterEngine class]);
+}
+
+- (void)tearDown {
+  [self.mockEngine stopMocking];
```

Comment:
Can you add a comment about why we're doing this - especially since we're doing it for this mock and not others?

Reply:
Done.